autopilot/interface.go

```diff
@@ -182,6 +206,11 @@ func init() {
 	for _, h := range availableHeuristics {
 		AvailableHeuristics[h.Name()] = h
 	}
+
+	// Fill available node metrics.
+	for _, h := range availableNodeMetrics {
+		AvailableNodeMetrics[h.Name()] = h
```

Comment:
should we check for overwrites, here and above?

Reply:
Dunno, I feel like this should be more of a static assert if we'd have something like that, otherwise maybe review time assert :)